src/kernel-loader/index.ts

```diff
@@ -256,10 +256,8 @@ async function initKernel() {
     store.dispatch(setKernelError(error))
 
     // TODO: move this into a saga for setKernelError
-    trackCriticalError(error.error, error.extra)
-    if (error.level === 'fatal') {
-      disableAnalytics()
-    }
+    trackError(error.error, { context: 'kernel', ...(error.extra || {}) })
+    disableAnalytics()
```

Comment:
Should we need to disable analytics at any error level?

Reply:
Yep, since the error produces a black screen of death. Will add a comment